nucleus/__init__.py

```diff
@@ -961,6 +975,42 @@ def delete_slice(self, slice_id: str) -> dict:
         )
         return response
 
+    def append_to_slice(
```

Comment:
I moved this here to stop pylint complaining about directly accessing the client's private method